docs/platform/stackscripts/creating-and-managing-stackscripts-a-tutorial/index.md

```diff
@@ -0,0 +1,156 @@
+---
+author:
+  name: Linode
+  email: docs@linode.com
+description: 'This guide shows you how to create a Linode StackScript using the Linode Cloud Manager. You will also learn how to perform various tasks, like editing an Account StackScript, making an Account StackScript public, and deleting an Account StackScript. '
+og_description: 'This guide shows you how to create a Linode StackScript using the Linode Cloud Manager. You will also learn how to perform various tasks, like editing an Account StackScript, making an Account StackScript public, and deleting an Account StackScript.'
+keywords: ["automation", "scripts", "deployments", "instance"]
+license: '[CC BY-ND 4.0](https://creativecommons.org/licenses/by-nd/4.0)'
+modified: 2020-04-22
+modified_by:
+  name: Linode
+published: 2020-04-22
+title: Create and Manage StackScripts - A Tutorial
+h1_title: A Tutorial for Creating and Managing StackScripts
+---
+
+## What are StackScripts?
+
+[StackScripts](http://linode.com/stackscripts/) provide Linode users with the ability to automate the deployment of custom systems on top Linode's default Linux distribution images. For example, every time you deploy a new Linode you might execute the same tasks, like updating your system's software, installing your favorite Linux tools, and adding a limited user account. These tasks can be automated using a StackScript that will perform these actions for you as part of your Linode's first boot process.
```

Comment:
The opening section for each of these guides is the same, with "What are StackScripts?" That might hurt SEO, and from a user experience perspective it might be difficult to readily distinguish between content. Maybe we could bring the In This Guide section up to the top to help immediacy? Open to suggestions.

Reply:
Andy and I discussed this and checked with Tyler to ensure that the meta descriptions and titles provided enough SEO bang to keep the intro the same. We will test this to ensure this decision makes sense. Keeping as is for now.